core/src/test/java/org/infinispan/topology/TestClusterCacheStatus.java

```diff
@@ -135,7 +135,7 @@ public void startConflictResolution(ConsistentHash conflictCH, Address... mergeM
 
    public void startConflictResolution(ConsistentHash conflictCH, List<Address> mergeMembers) {
       topology = new CacheTopology(topology.getTopologyId() + 1, topology.getRebalanceId() + 1,
-                                   readConsistentHash(), conflictCH, conflictCH,
+                                   conflictCH, conflictCH, conflictCH,
```

Comment:
Ok, so removing only the union CH doesn't work, but now that I got you to use the conflict CH for reading, how about we only set the `currentCH` and change `CacheTopology` to use it for everything during `CONFLICT_RESOLUTION`?

Reply:
I was mistaken here, this still needs to be the readconsistent hash after all as this is how we determine the preferred partition in the ConflictManager.

Reply:
That would undo the ISPN-9032 fix... maybe we could set the preferred read CH as the pendingCH for internal use, but use the currentCH (i.e. conflict CH) for both reading and writing?

Or we could just leave ISPN-9032 out of this PR and change the test to block the installation of the CONFLICT_RESOLUTION topology :)

Reply:
What I meant about still needing the currentCH was that atm we call `getCurrentCH` to determine the preferred partition in the conflict manager. But this is hacky anyway, so I'm going to change `resolveConlicts(topology)` to `resolveConflicts(topology, preferredOwners)` and then we can just use the `currentCH` as you suggested.

Reply:
Ok, got it now